go/exocom/features/configuring-the-ports.feature

```diff
@@ -0,0 +1,24 @@
+Feature: configuring the port
+
+  As an Exosphere developer testing my application
+  I want to be able to boot ExoCom at various ports
```

Comment:
I don't think we need this feature anymore now that Exocom runs inside Docker. We can always run it at a certain port like 3100.